onset_maproom/widgets.py

```diff
@@ -50,12 +50,12 @@ def Sentence(*elems):
         groups.extend(elems[0])
 
     for i in range(start, len(elems) - (1 if tail else 0), 2):
-        assert isinstance(elems[i], str)
+        assert (isinstance(elems[i], str) or isinstance(elems[i], html.Span))
```

Comment:
@kgraaf could you comment here? I'm not sure I understand the intent of the original assertion so I don't know if it's appropriate to relax it like this.

Reply:
@kgraaf please comment.

Reply:
make it an issue?

Reply:
We have bigger fish to fry.